src/graphnet/data/i3extractor.py

```diff
@@ -304,7 +304,29 @@ def __call__(self, frame, padding_value=-1) -> dict:
                     'final_position_y': final_position[1],
                     'final_position_z': final_position[2],
                     'stopped_muon': stopped,
-                })                
+                })  
+            if frame_contains_retro(frame):
+                if abs(MCInIcePrimary.pdg_encoding) != 13: # if not muon
+                    output.update({
+                        'osc_weight'         : frame["I3MCWeightDict"]["weight"],
+                        'OneWeight'          : frame["I3MCWeightDict"]["OneWeight"],
```

Comment:
I would suggest that we stick to snake-case naming convention for attributes (i.e. `OneWeight` -> `one_weight` and `NEvents` -> `n_events`) for consistency.

Reply:
Done @asogaard